lib/Cake/Test/Case/Utility/ValidationTest.php

```diff
@@ -148,6 +148,10 @@ public function tearDown() {
  * @return void
  */
 	public function testNotBlank() {
+		$this->assertTrue(Validation::notBlank(0));
+		$this->assertTrue(Validation::notBlank(0.0));
+		$this->assertTrue(Validation::notBlank(-0));
+		$this->assertTrue(Validation::notBlank(-0.0));
```

Comment:
Looks like the behavior of `-0.0` became correct in PHP7+, but in PHP5 `0.0 * -1` was required. See https://3v4l.org/a0FSJ